src/bootstrap/configure.py

```diff
@@ -279,6 +279,10 @@ def build():
 
 
 def set(key, value):
+    if isinstance(value, list):
+        # Remove empty values, which value.split(',') tends to generate.
+        value = [v for v in value if v]
```

Comment:
Huh! I have never seen the `if v` modifier on this kind of loop. Do you know if that's a Python 3 addition?

I'm not sure if this is always a result of `split(',')`, but it seems like it's probably mostly always OK to drop these from the list if they get generated.

Reply:
My understanding is that empty strings are always considered false-y in Python, both 2 & 3. See https://docs.python.org/2/library/stdtypes.html#truth-value-testing and https://docs.python.org/3/library/stdtypes.html#truth-value-testing

Reply:
the `if` clause on list comprehensions has been supported for as long as list comprehensions have been in python (since 2.0)

Reply:
Thanks!